app/javascript/packages/document-capture/components/location-collection-item.tsx

```diff
@@ -2,33 +2,47 @@ import { Button } from '@18f/identity-components';
 import { useI18n } from '@18f/identity-react-i18n';
 
 interface LocationCollectionItemProps {
+  distance?: string;
   formattedCityStateZip: string;
   handleSelect: (event: React.FormEvent<HTMLInputElement>, selection: number) => void;
-  name: string;
+  name?: string;
+  phone?: string;
   saturdayHours: string;
   selectId: number;
   streetAddress: string;
   sundayHours: string;
+  tty?: string;
   weekdayHours: string;
 }
 
 function LocationCollectionItem({
+  distance,
   formattedCityStateZip,
   handleSelect,
   name,
+  phone,
   saturdayHours,
   selectId,
   streetAddress,
   sundayHours,
+  tty,
   weekdayHours,
 }: LocationCollectionItemProps) {
   const { t } = useI18n();
+  const numericDistance = distance?.split(' ');
```

Comment:
This is kind of confusing because `numericDistance` will have a value like `['0.1', 'mi']` which isn't numeric. Would prefer pulling out the numeric piece here instead of in the rendering logic

```suggestion
  // distance is a string like '0.1 mi'. Split out the numeric piece
  const numericDistance = distance?.split(' ')[0];
```

Reply:
good point